gdnative_api.json

```diff
@@ -6304,7 +6304,7 @@
       "type": "ANDROID",
       "version": {
         "major": 1,
-        "minor": 0
```

Comment:
This seems like an error to me since now there is no `1.0` accessible anymore. The added functions should have been in a new separate extension if the `1.0` was already shipped in Godot at some point.

Reply:
True, I'm not to fussed by it though, the original calls were made for the original oculus gearvr plugin which never worked to begin with, so it is unlikely anyone was actually using this extension.